fews_3di/tests/example_settings.xml

```diff
@@ -22,5 +22,7 @@
         <string key="simulationname" value="Simulation name"/>
         <string key="save_state" value="True"/>
         <string key="saved_state_expiry_days" value="5"/>
+        <string key="process_basic_results" value="True"/>
+        <string key="results_scenario_name" value="ivar test"/>
```

Comment:
Die results_scenario_name is optioneel, maar je maakt er wel gebruik van. Dat klopt niet met elkaar.

Is dit iets waarvoor er ook echt een aparte setting moet komen? Ik voeg niet graag settings toe, want dan moeten alle configs aangepast worden.

Is het niet veel makkelijker om een goede waarde hiervoor te genereren? Je hebt "simulationname" al, dus lijkt het me prima om dat te gebruiken plus dan " results" erachter geplakt ofzo.

Reply:
Reden achter deze opzet:
De results_scenario_name is noodzakelijk wanneer process_basic_results aan staat. Process_basic_results is een boolean die aangeeft of resultaten wel of niet moeten worden naverwerkt in Lizard. Het is belangrijk om de naam zelf te kunnen kiezen, zo is er de mogelijkheid om over oude resultaten heen te schrijven (en dus een operationeel live beeld in lizard te tonen) of juist beelden met een timestamp te bewaren of scenario's te draaien. 

Hoe zou jij de optionele property meenemen (als je geen kale try except wilt)